cmd/sandbox/start.go

```diff
@@ -136,36 +128,24 @@ func startSandbox(ctx context.Context, cli docker.Docker, reader io.Reader) (*bu
 	} else if vol != nil {
 		volumes = append(volumes, *vol)
 	}
-
-	if len(sandboxConfig.DefaultConfig.Version) > 0 {
-		isGreater, err := util.IsVersionGreaterThan(sandboxConfig.DefaultConfig.Version, flyteMinimumVersionSupported)
+	if sandboxConfig.DefaultConfig.Version != flyteTag {
+		sha, err := githubutil.GetSHAFromVersion(sandboxConfig.DefaultConfig.Version, "flyte")
 		if err != nil {
 			return nil, err
 		}
-		if !isGreater {
-			logger.Infof(ctx, "version flag only supported after with flyte %s+ release", flyteMinimumVersionSupported)
-			return nil, fmt.Errorf("version flag only supported after with flyte %s+ release", flyteMinimumVersionSupported)
-		}
-		if err := githubutil.GetFlyteManifest(sandboxConfig.DefaultConfig.Version, flyteManifest); err != nil {
-			return nil, err
-		}
-
-		if vol, err := mountVolume(flyteManifest, generatedManifest); err != nil {
-			return nil, err
-		} else if vol != nil {
-			volumes = append(volumes, *vol)
-		}
-
+		sandboxConfig.DefaultConfig.Version = fmt.Sprintf("%s-%s", flyteTag, sha)
 	}
 
-	fmt.Printf("%v pulling docker image %s\n", emoji.Whale, docker.ImageName)
-	if err := docker.PullDockerImage(ctx, cli, docker.ImageName); err != nil {
+	var sandboxImageName = fmt.Sprintf("%s:%s", docker.ImageName, sandboxConfig.DefaultConfig.Version)
```

Comment:
define this format before using it here.
And also can we mention an example in the comments

Reply:
I dont see a change here. Hence unresolving